src/components/pages/metadata-api/metadata-api-presentation.tsx

```diff
@@ -0,0 +1,337 @@
+            )}
+          >
+            <span className="underlined !pr-0 pb-2">Metadata API</span>
+          </h1>
+        </section>
+
+        <section
+          className={clsx('text-left mt-16 mb-16 px-4 mx-auto max-w-5xl')}
+        >
+          <div className="sm:flex sm:justify-between">
+            <div className="sm:flex-1 text-xl leading-cozy sm:max-w-[31rem] sm:flex sm:flex-col pr-2">
+              <h2 className="text-gray-700 text-[1.3rem] font-extrabold tracking-tight">
+                <br />
+                Integrate our Digital Learning Resources into your apps
+              </h2>
+              <p className="mt-4">
+                Our new Metadata API makes it easy to integrate thousands of
+                Serlo articles, courses, videos, and interactive exercises. Just
+                read on and take a look at the codesnippets below to get
+                started.
+              </p>
+              <p className="mt-4">
+                Our metadata API is completely free and it only takes a few
+                lines of code to integrate a vast repository of educational
+                resources into your app!
+              </p>
+            </div>
+            <div className="sm:flex-1 mt-8 ml-4 -mx-side sm:max-w-[32rem] rounded-lg overflow-hidden">
+              <img
+                src="/_assets/img/metadata/metadata.png"
+                alt="A bird flying over a stack of documents"
+              />
+            </div>
+          </div>
+        </section>
+
+        <section
+          className={clsx('bg-orangeBow bg-100% px-2 mt-0 !pt-16 mb-20')}
+        >
+          <div className="mt-2 pb-16 sm:flex text-center text-xl max-w-4xl mx-auto px-4">
+            <div className="flex-1 mt-5">
+              <b className="text-brand font-handwritten text-4xl">20,000+</b>
+              <br />
+              Educational resources have been created with the{' '}
+              <Link href="https://serlo.org/editor">Serlo Editor</Link>
+            </div>
+            <div className="flex-1 mt-5">
+              <b className="text-brand font-handwritten text-4xl">1 Mio+</b>
+              <br />
+              Users per month already learn with Serlo
+            </div>
+          </div>
+        </section>
+
+        <section className={clsx('mt-24 pb-16')}>
+          <div className="text-3xl leading-cozy max-w-4xl text-center mx-auto"></div>
+
+          <div className="mt-12 text-3xl leading-cozy max-w-4xl text-center mx-auto">
+            <h2 className="text-center text-4xl leading-cozy tracking-tight font-extrabold mb-4">
+              Development and license
+            </h2>
+            <div className="text-left mx-side">
+              <p className="mt-4 text-xl leading-cozy flex-1">
+                Join our growing community by integrating our content through
+                the Serlo Metadata API into your apps and change the future of
+                education with us!
```

Comment:
I think it may confuse readers, that may think that it is like our content API.

```suggestion
                Find more easily our contents using Serlo Metadata API!
```